webpack.config.js

```diff
@@ -0,0 +1,53 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+
+module.exports = {
+  mode: 'development',
+  entry: './amt/site/static/ts/amt.ts',
+  output: {
+    filename: 'amt.js',
+    path: path.resolve(__dirname, 'amt/site/static/dist'),
```

Comment:
I read it is wise to add 'clean: true' so the dist folders is cleared before building.

```suggestion
    path: path.resolve(__dirname, 'amt/site/static/dist'),
    clean: true
```